### DISCUSSION TITLE
How to get basic "beeps" working from PC-speaker?

### DISCUSSION THREAD

**Discussion by jharrison03** [dosemu2/dosemu2#1938] [General]
Just the beeps from a PC speaker (not sound card stuff). Nothing more elaborate than that.  I've tried $_speaker = "emulated" and "native" and neither seems to work.

I would expect this to come through the sound card but the actual PC speaker would be OK too.

**Comment by stsp** [comment]
Unfortunately this is not implemented "yet".
But as a work-around you can try
`$_speaker="native"` after which, run
`dosemu -s`
That may require configuring sudo, and
is overall not recommended to do. But so
far this is the only way to get the speaker
working under dosemu2. :(
